pkg/render/monitor/monitor.go

```diff
@@ -950,10 +956,10 @@ func (mc *monitorComponent) serviceMonitorQueryServer() *monitoringv1.ServiceMon
 	}
 }
 
-func (mc *monitorComponent) operatorRole() *rbacv1.Role {
+func (mc *monitorComponent) operatorRole() []*rbacv1.Role {
 	// list and watch have to be cluster scopes for watches to work.
 	// In controller-runtime, watches are by default non-namespaced.
-	return &rbacv1.Role{
```

Comment:
I don't think there is a need to create variables, a list can also be created in-line.